.github/workflows/ci.yml

```diff
@@ -81,8 +82,6 @@ jobs:
         run: make html
       - name: Test doctests in docstrings
         working-directory: doc
-        env:
-          NBSPHINX_EXECUTE: YES
```

Comment:
why can this be removed now?

Reply:
The doctest job should only check of `doctest` directives in the docstrings. Now it also runs the notebooks, which makes the workflow longer by some two minutes.

Note that the notebooks are already run in the job prior to it:
https://github.com/ComPWA/expertsystem/pull/289/files/7d18e93af3011db9576dc56a5a388b2e59e86400#diff-e9f950f17198d3d5e3122a44230a09b9R78-R82

Reply:
i see